std/experimental/xml/appender.d

```diff
@@ -0,0 +1,198 @@
+/*
+*             Copyright Lodovico Giaretta 2016 - .
+*  Distributed under the Boost Software License, Version 1.0.
+*      (See accompanying file LICENSE_1_0.txt or copy at
+*            http://www.boost.org/LICENSE_1_0.txt)
+*/
+
+/*
+*   Appender implementation that uses a custom allocator. Meant for internal usage.
+*/
+
+module std.experimental.xml.appender;
+
+/*
+*   Appender implementation that uses a custom allocator. Meant for internal usage.
+*/
+struct Appender(T, Alloc)
```

Comment:
You are right. This is not the only place where `package` is missing. When I tried to add it, I found a DMD bug. When the bug got fixed, I forgot to fix my code.